zipline/data/data_portal.py

```diff
@@ -495,6 +501,15 @@ def __init__(self,
             'price': None
         }
         self._equity_daily_reader_array_data = {}
+        self._equity_minute_loader_array_keys = {
```

Comment:
can you add a comment explaining what's going to be stored in `self._equity_minute_loader_array_keys` and `self._equity_minute_loader_array_data`?  I assume it's cached data of some sort, but hard to tell.